.github/quarkusbuilditemdoc.java

```diff
@@ -93,7 +95,12 @@ private String getJavaDoc(JavaDocCapable<?> source) {
             Files.walkFileTree(path, new SimpleFileVisitor<Path>() {
                 @Override
                 public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
-                    if (file.toString().endsWith("BuildItem.java")) {
+                    String fileContent = Files.readString(file, StandardCharsets.ISO_8859_1);
+                    List<String> keywordList = Arrays.asList("extends BuildItem {", "extends SimpleBuildItem {",
+                            "extends MultiBuildItem {", "extends EmptyBuildItem {");
+                    boolean match = keywordList.stream().anyMatch(fileContent::contains);
+                    if (file.toString().endsWith("BuildItem.java")) {            //original version
+                    //if (file.toString().endsWith("BuildItem.java") || match) {  // edited version to include more build items
                         process(multimap, file);
```

Comment:
Since the file is already read in memory, you can pass those contents to the `Roaster.parse` method